packages/experimental/mosaic/src/components/ScrollContainer/ScrollContainer.tsx

```diff
@@ -0,0 +1,96 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import * as ScrollArea from '@radix-ui/react-scroll-area';
+import React, { ReactNode } from 'react';
+
+import { mx } from '@dxos/react-components';
+
+export type ScrollContainerSlots = {
+  root?: {
+    className?: string;
+  };
+  viewport?: {
+    className?: string;
+  };
+  scrollbar?: {
+    className?: string;
+  };
+  thumb?: {
+    className?: string;
+  };
+};
+
+// TODO(burdon): System colors.
+// TODO(burdon): Is this right? Merge?
+export const defaultSlots: ScrollContainerSlots = {
+  scrollbar: {
+    className: 'bg-neutral-50 hover:bg-neutral-100'
+  },
+  thumb: {
+    className: 'bg-neutral-400'
+  }
+};
```

Comment:
I’d prefer to put these in a `scrollContainerSyles.ts` file if other components might need them, otherwise just use them as literals in the component itself.